templates/common/_base/units/mtu-migration.service.yaml

```diff
@@ -0,0 +1,19 @@
+name: mtu-migration.service
```

Comment:
We only need to render this service if Migration is enabled. You can look for tips at
https://github.com/openshift/machine-config-operator/blob/2c75679cf5a1c09f4040ad678ff1ebe709797766/templates/common/_base/files/mtu-migration-conf.yaml